src/main/java/io/cdap/e2e/pages/locators/CdfControlCenterLocators.java

```diff
@@ -0,0 +1,167 @@
+  public static WebElement navigateOnViewDetailsPageDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Back')]")
+  public static WebElement clickOnBackTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//*[@class='icon-svg icon-close']")
+  public static WebElement clickOnCloseTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying all entities, sorted by Newest')]")
+  public static WebElement clickOnDatasetApplicationsArtifactsMessage;
+
+  public static By locateDropdownListItem(String option) {
+    return By.xpath("//input[@data-cy='" + option + "']");
+  }
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Schema')]")
+  public static WebElement clickOnSchemaTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Programs')]")
+  public static WebElement clickOnProgramsTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='schema-tab']")
+  public static WebElement clickOnSchemaTabDatasetEntityPage;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='program-tab clearfix']")
+  public static WebElement clickOnProgramsTabDatasetEntityPage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Newest')]")
+  public static WebElement clickOnNewestOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Oldest')]")
+  public static WebElement clickOnOldestOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Z-A')]")
+  public static WebElement clickOnZToAOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by A-Z')]")
+  public static WebElement clickOnAToZOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//div[@id='filter-tooltip-target-id']")
+  public static WebElement clickOnSortDropdown;
+
+  public static WebElement selectSortDropdownValue(String option) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//*[contains(text(),'" + option + "')]"));
+  }
+
+  @FindBy(how = How.XPATH, using = "//span[@data-testid='tag-plus-button']")
+  public static WebElement clickOnAddTagIconDatasetEntity;
+
+  public static WebElement enterAddTagIconDatasetEntityValue() {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@data-testid='tag-input']"));
+  }
+
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'Tags (1)')]")
+  public static WebElement tagCountIconIncreaseDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[@class='tag-content']//*[@class='icon-svg icon-close']")
+  public static WebElement clickOnCloseTagIconDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//i[normalize-space()='No tags found. Click to add a new business tag.']")
+  public static WebElement tagCounDecreasetIconDatasetEntity;
```

Comment:
why this name for the locator . we are just seeing a message in this locator , change it to defaultTagCountMessageDataset or something like that

Reply:
we are removing the tag , here so we are writing decreease intially it was one and then removing the added

Reply:
Changed the locator name and included the word message